source/tutorials/kubernetes.rst

```diff
@@ -3,354 +3,657 @@
-:abbr:`OCI (Open Container Initiative*)` guidelines and works seamlessly with
-Kubernetes. `Kata Containers`_ provides strong isolation for untrusted
-workloads or  multi-tenant scenarios. Kata Containers can be
-allocated on a per-pod basis, so you can mix and match both on the same host
-to suit your needs.
+  * `What is containerd?`_
+
+  * `What is CRI+O?`_
+
+* `What is Kata Containers?`_
 
 Prerequisites
 *************
 
 This tutorial assumes you have already installed |CL|. For detailed
-instructions on installing |CL| on a bare metal system, follow the
-:ref:`bare metal installation tutorial<bare-metal-install-desktop>`.
-Learn about the benefits of having an up-to-date system for cloud
-orchestration on the :ref:`swupd-guide` page.
+instructions on installing |CL| on a bare metal system, follow the :ref:`bare
+metal installation tutorial<bare-metal-install-desktop>`.
 
-Before you install any new packages, update |CL| with the following command:
+#. Before you continue, update your |CL| installation with the following command:
 
-.. code-block:: bash
+   .. code-block:: bash
 
-   sudo swupd update
+      sudo swupd update
 
-Install Kubernetes and CRI runtimes
-***********************************
+   Learn about the benefits of having an up-to-date system for cloud
+   orchestration on the :ref:`swupd-guide` page.
 
-Kubernetes, a set of supported :abbr:`CRI (Container Runtime Interface)`
-runtimes, and networking plugins, are included in the `cloud-native-basic`_
-bundle.
+#. Kubernetes, a set of supported :abbr:`CRI (Container Runtime Interface)`
+   runtimes, :abbr:`CNI (Container Network Interface)` and `cloud-native-setup
+   scripts`_ are included in the `cloud-native-basic`_ bundle. Install the
+   cloud-native-basic bundle to get these components:
 
-To install this framework, enter the following command:
+   .. code-block:: bash
 
-.. code-block:: bash
+      sudo swupd bundle-add cloud-native-basic
 
-   sudo swupd bundle-add cloud-native-basic
+
+Set up Kubernetes automatically
+*******************************
+
+|CL| provides `cloud-native-setup scripts`_ to automate system setup and
+Kubernetes cluster initialization which allows you to get a cluster up and
+running quickly.
 
 .. note::
 
-   For more on networking plugins, see `Install pod network add-on`_.
+   By default, the :file:`create_stack.sh` script setups Kubernetes with
+   **canal for container networking** and **crio for container runtime**.
+   The script can be configured to use other CNI and CRI by following the
+   directions on the `README
+   <https://github.com/clearlinux/cloud-native-setup/blob/master/clr-k8s-examples/README.md>`_.
+   
+   See `What is a Container Network Interface (CNI)?`_ and `What is a
+   Container Runtime Interface (CRI)?`_ for more information.
+
+.. important::
+
+   If any network proxy settings are required for Internet connectivity
+   configure them now because the scripts will propagate proxy configuration
```

Comment:
I would recommend that people put their proxy info in /etc/profile.d/proxy.sh as that is one place where the script looks for proxy information.  The /etc/environment is the other place but I would recommend they pick one. I always put in /etc/profile.d/proxy.sh

Reply:
I'm not going to address this in this PR. The linked proxy documentation does explain this.